examples/cpp/route_guide/helper.cc

```diff
@@ -43,6 +44,7 @@ ABSL_FLAG(std::string, db_path, "route_guide_db.json", "Path to db file");
 namespace routeguide {
 
 std::string GetDbFileContent(int argc, char** argv) {
+  absl::ParseCommandLine(argc, argv);
```

Comment:
This is something that should be in the `main()`, as it is also needed to initialize some other subsystems, like logging. Please confirm it is not already called from there.